Green-Kittens/bathroom-finder-frontend
# Implementation: Image Carousel on Facility Profile

# Checklist

- [x] Have I clearly explained the issue or feature request that this PR addresses?
- [x] Have I provided where new implementations for the PR have been made?
- [x] Did I highlight any areas that may need special attention during the review?
- [x] If applicable, did I specify any setup requirements or dependencies for testing?
- [x] Have I disclosed any known issues or limitations related to this PR? Did I explain why these limitations exist and how they might be addressed in the future?
- [x] Have I provided enough context for someone unfamiliar with the issue or the codebase to understand the changes?
- [x] Have I assigned appropriate reviewers who are knowledgeable about the changes? Did I consider including team members who might be affected or interested in the changes?

# Description

## Summary
Issue #140:
- originally the facility profile screen image display was for a single image. Since we allow a user to upload multiple images to a facility on the facility form, this new implementation supports the display of all images uploaded to the facility. 

## Expectations
- files changed: facility-profile.tsx
- steps for testing: 
   1. navigate to a facility profile 
   2. see that the image display is now in a image carousel (visually, the image carousel should look very similar to the one implemented in the facility and review forms-- only difference is that there is no option to delete an image, like there is on the facility and review forms when you click on the image)
- upon clicking on an image in the carousel, expect to see a modal pop up of the image with an "x" button on the top right corner

## Additional Comments/Notes
- did not use Carousel.tsx component as user should not be able to delete an image from the carousel
